projects.md

```diff
@@ -0,0 +1,45 @@
+# Projects
+
+## Sales leads & internal proposals
+
+While the Business Planning WG is mainly in charge of sales lead generation, all members may pursue leads as they come up. These may be a client or a grant opportunity. Members are strongly encouraged to record leads into the co-operative's [leads tracking sheet](https://link.hypha.coop/leads) whether or not we decide to pursue the opportunity.
+
+Members leading a sales effort should draft an **internal proposal** and present it at a meeting to see whether the co-operative wants to move forward to submit a proposal, whether there are concerns by other members, and gauge team capacity given project timelines. A sample internal proposal should include:
+
+```
+# Project title
+
+## Background
+Background information about the client/grant and the project.
+
+## Scope & timeline
+What are the expected deliverables and the timeline.
+
+## Outcomes & opportunities
+How this project may be beneficial beyond the scope of the project. For example, does it lead to developing new skillsets among members or starting a relationship with a potentially long-term client?
+
+## Skillsets
+What are the important skillsets needed to ship this. If some skillsets are not available, what are the plans to source them to ensure we can deliver on this?
+
+## Budget
+Project budget and payment terms if available.
+
+## Communication
+How the logistics would look like for members working on this project. Are there weekly calls or are there physical sessions? Is the project on GitHub or is the source closed? Are there consent and confidentiality terms?
+
+## Risks
+What are the expected risks associated with this project.
+
+## Resources
+Additional resources about the project not already listed above, such as link to a website or a call for proposal.
+```
+
+You can add more sections as appropriate for the project. This may include sample code, project roadmap, next steps, etc. You can find examples in our archived proposals folder in our [shared drive](http://link.hypha.coop/drive).
+
+## Proposal drafting and submissions
```

Comment:
This needs a different term if you've called an "internal proposal" above, "project brief"? "grant proposal"?

Reply:
Client proposal?